doc/api/net.md

```diff
@@ -246,168 +354,215 @@ server.on('error', (e) => {
 });
 ```
 
-(Note: All sockets in Node.js are set `SO_REUSEADDR`.)
+*Note*: All `net.Sockets` in Node.js are set to use the `SO_REUSEADDR` flag
```

Comment:
I think the backticks should not include the `s` here as in `net.Socket`s and not `net.Sockets`